config/00-init/02-db-pvc.yaml

```diff
@@ -15,9 +15,9 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: db
+  name: tekton-hub-db
   labels:
-    app: db
+    app: tekton-hub-db
```

Comment:
do we want to change the value of the labels as well?
cc @PuneetPunamiya @piyush-garg @sm43